automation/rancher-partner-charts/questions.yaml

```diff
@@ -862,7 +862,7 @@ questions:
 - variable: auth-server.image.tag
   required: true
   type: string
-  default: "1.1.5_dev"
+  default: "1.1.5"
```

Comment:
Image tag should be pointed to `1.1.5-1` as there's no `1.1.5` tag in Janssen packages.